packages/Main/src/Layer/CopcLayer.js

```diff
@@ -37,22 +37,24 @@ class CopcLayer extends PointCloudLayer {
          */
         this.isCopcLayer = true;
 
-        const resolve = () => this;
+        const resolve = () => PointCloudLayer.prototype.addInitializationStep.call(this);
```

Comment:
could be replaced by `super.addInitializationStep();`

Reply:
Done